docs/src/use/troubleshooting/couldnt-find-the-plugin.md

```diff
@@ -30,17 +22,40 @@ It's likely that the plugin isn't installed correctly. Try reinstalling by runni
 The plugin "${pluginName}" was referenced from the config file in "${importerName}".
 ```
 
+## Cause
+
+[Legacy ESLint configuration files](../configure/configuration-files-deprecated) specify shareable configs by their package name.
+That package name is passed to the Node.js `require()`, which looks up the package under local `node_modules/` directories.
+For example, the following ESLint config will first try to load a module located at `node_modules/eslint-plugin-yours`:
+
+```js
+module.exports = {
+    extends: ["eslint-plugin-yours"],
```

Comment:
Did you mean `plugins` here?

This would need to be something like this load a config:
```suggestion
    extends: ["plugin:eslint-plugin-yours/config-name"],
```